h/claim/invite.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+"""
+:mod:`h.claim.invite` is a utility to invite users to claim accounts
+and is exposed as the command-line utility hypothesis-invite.
+"""
+import argparse
+import logging
+import os
+import sys
+import time
+
+import mandrill
```

Comment:
The build is complaining about this. You can either add the extra to `.travis.yml` or `# noqa` this. Probably the former.